<?xml version="1.0" encoding="UTF-8"?><testrun name="CashCardJsonTest (1)" project="cashcard" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="springapp.cashcard.CashCardJsonTest" time="0.0" displayname="CashCardJsonTest" uniqueid="[engine:junit-jupiter]/[class:springapp.cashcard.CashCardJsonTest]">
    <testcase name="myFirstTest" classname="springapp.cashcard.CashCardJsonTest" time="0.0" displayname="myFirstTest()" uniqueid="[engine:junit-jupiter]/[class:springapp.cashcard.CashCardJsonTest]/[method:myFirstTest()]">
      <failure>
        <expected>42</expected>
        <actual>1</actual>
        org.opentest4j.AssertionFailedError: &#13;
expected: 42&#13;
 but was: 1&#13;
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)&#13;
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#13;
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)&#13;
	at springapp.cashcard.CashCardJsonTest.myFirstTest(CashCardJsonTest.java:11)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;

      </failure>
    </testcase>
  </testsuite>
</testrun>
